app/src/main/java/io/apicurio/registry/rest/RegistryApplicationServletFilter.java

```diff
@@ -75,6 +72,7 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
         HttpServletRequest req = (HttpServletRequest) request;
         String requestURI = req.getRequestURI();
         if (requestURI != null) {
+            //TODO ensure tenant is authenticated at this point, because tenantIdResolver will fetch tenant's configuration from tenant-manager
```

Comment:
This is the point where we'll need to check if the tenant belongs to a given org, good.